students/pkoleyni/Lesson03/Slicing_Lab.py

```diff
@@ -0,0 +1,49 @@
+def exchange_first_last(seq):
+    """
+    :param seq: any python seq, like String,List,Tuple
+    :return:
+    """
+    return (seq[-1:]+seq[1:-1]+seq[0])
```

Comment:
this would give you an error with a list or a tuple:
```
>>> seq = [1, 2, 3, 4, 5, 6]
>>> seq[-1:]+seq[1:-1]+seq[0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate list (not "int") to list
```

remember that you should **_only_** use slicing; `seq[0]` is called indexing